chrome/content/zotfile/zotfile.js

```diff
@@ -1926,7 +1926,14 @@ Zotero.ZotFile = {
               // create a nslFile Object of the destination folder
               var dir = this.createFile(destination);
               // move file to new location
-              file.moveTo(dir, filename);
+              if(destination.trim() == "/") { 
```

Comment:
When exactly is `destination.trim()` equal to `"/"`? What is with Windows vs Mac?